openslides_backend/action/action.py

```diff
@@ -715,6 +715,17 @@ def get_on_failure(self, action_data: ActionData) -> Optional[Callable[[], None]
         """
         return None
 
+    def execute_presenter(self, PresenterClass: Any, payload: Any) -> Any:
```

Comment:
```suggestion
    def execute_presenter(self, PresenterClass: Type[BasePresenter], payload: Any) -> Any:
```

Reply:
I cannot import `BasePresenter` in the Action class, because of circular imports. So it won't work.

Reply:
I get:
`ImportError: cannot import name 'Action' from partially initialized module 'openslides_backend.action.action' (most likely due to a circular import) (/app/openslides_backend/action/action.py)`

Reply:
Move the `export_helper.py` file into the `shared` folder - it is used in actions and presenters and does not belong in the action folder. This should resolve the circular import.

Reply:
Moved export_helper into the shared folder. Add type BasePresenter to PresenterClass.